packages/@react-spectrum/s2/src/Tabs.tsx

```diff
@@ -484,65 +513,59 @@ let HiddenTabs = function (props: {
   );
 };
 
-let TabsMenu = (props: {items: Array<Node<any>>, onSelectionChange: TabsProps['onSelectionChange']}) => {
-  let stringFormatter = useLocalizedStringFormatter(intlMessages, '@react-spectrum/s2');
-  let {items} = props;
+let TabsMenu = (props: {items: Array<Node<any>>, onSelectionChange: TabsProps['onSelectionChange']} & TabsProps) => {
+  let {id, items} = props;
   let {density, onSelectionChange, selectedKey, isDisabled, disabledKeys, pickerRef, labelBehavior} = useContext(InternalTabsContext);
   let state = useContext(TabListStateContext);
   let allKeysDisabled = useMemo(() => {
     return isAllTabsDisabled(state?.collection, disabledKeys ? new Set(disabledKeys) : new Set());
   }, [state?.collection, disabledKeys]);
 
   return (
-    <UNSTABLE_CollectionRendererContext.Provider value={UNSTABLE_DefaultCollectionRenderer}>
-      <div
-        className={style({
-          display: 'flex',
-          alignItems: 'center',
-          height: {
-            density: {
-              compact: 32,
-              regular: 48
-            }
-          }})({density})}>
-        <Picker
-          ref={pickerRef ? pickerRef : undefined}
-          isDisabled={isDisabled || allKeysDisabled}
-          density={density!}
-          labelBehavior={labelBehavior}
-          items={items}
-          disabledKeys={disabledKeys}
-          selectedKey={selectedKey}
-          onSelectionChange={onSelectionChange}
-          aria-label={stringFormatter.format('tabs.selectorLabel')}>
-          {(item: Node<any>) => {
-            // need to determine the best way to handle icon only -> icon and text
-            // good enough to aria-label the picker item?
-            return (
-              <PickerItem
-                {...item.props.originalProps}
-                isDisabled={isDisabled || allKeysDisabled}
-                key={item.key}>
-                {item.props.children({density, isMenu: true})}
-              </PickerItem>
-            );
-          }}
-        </Picker>
-      </div>
-    </UNSTABLE_CollectionRendererContext.Provider>
+    <div
+      className={style({
+        display: 'flex',
+        alignItems: 'center',
+        height: {
+          density: {
+            compact: 32,
+            regular: 48
+          }
+        }})({density})}>
+      <Picker
+        id={id}
+        aria-label={props['aria-label']}
+        aria-labelledby={props['aria-labelledby']}
+        aria-describedby={props['aria-describedby']}
+        aria-details={props['aria-details']}
+        ref={pickerRef ? pickerRef : undefined}
+        isDisabled={isDisabled || allKeysDisabled}
+        density={density!}
+        labelBehavior={labelBehavior}
+        items={items}
+        disabledKeys={disabledKeys}
+        selectedKey={selectedKey}
+        onSelectionChange={onSelectionChange}>
+        {(item: Node<any>) => {
+          // need to determine the best way to handle icon only -> icon and text
```

Comment:
can remove this comment, I think we've addressed it and looks like it just got forgotten by me